available/reload_

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+# symlink this file to 'reload_<name of service to reload'
+
+filename="$(basename "$0")"
+service="${filename#*_}"
+echo "Reloading $service..."
+/usr/bin/sudo /bin/systemctl reload "$service"
```

Comment:
I really like this symlink scheme, it looks much nicer to work with than the older services-reload hook

It would be nice to avoid the hardcoded sudo, to work correctly on systems without sudo at all, or where policykit is configured to allow systemctl reload as a unprivileged user. How about a generic `sudo_` hook which works much like your `ifdomain_` hook, to allow running any other hook script under sudo?

Reply:
Good point, however I adopt a 'least privileges required' approach to security, so personally would not be happy running the entire script under sudo when the only command _required_ is "/bin/systemctl reload *".

I think it would be better to pull the systemctl command out as a variable that can be overridden in a config file (like the existing config hook) and defaulted in this file.

E.g.

```bash
if [ -z "$SYSTEMCTL_CMD" ]
then
    SYSTEMCTL_CMD=/bin/systemctl
fi
# ...
$SYSTEMCTL_CMD reload "$service"
```

Then, for my use-case:

```bash
echo 'export SYSTEMCTL_CMD="/usr/bin/sudo /bin/systemctl"' >> /etc/dehydrated/conf.d/systemctl_cmd.sh
```

Reply:
I've modified reload_ and docker-restart_ (which was also hardcoded to use sudo) to follow the pattern I just described.